intelmq/bots/BOTS

```diff
@@ -598,6 +605,17 @@
                 "hierarchical_output": false
             }
         },
+        "Files": {
+            "description": "Files is the bot responsible for lockfree saving of events into separate files.",
+            "module": "intelmq.bots.outputs.files.output",
+            "parameters": {
+                "tmp": "/opt/intelmq/var/lib/bots/files-output/tmp",
+                "dir": "/opt/intelmq/var/lib/bots/files-output/incoming",
+                "suffix": ".json",
+                "hierarchical_output": false,
+                "single_key": "output"
```

Comment:
I'm not sure if we want to have "output" as default, as opposed to all other output bots, which send/save the intelmq event as a whole. @aaronkaplan what do you think?